ports/stm32/qspi.c

```diff
@@ -346,6 +346,12 @@ STATIC void qspi_read_cmd_qaddr_qdata(void *self_in, uint8_t cmd, uint32_t addr,
     QUADSPI->ABR = 0; // alternate byte: disable continuous read mode
     QUADSPI->AR = addr; // address to read from
 
+    // Workaround for SR getting set immediately after setting the address.
+    if (QUADSPI->SR & 0x01) {
```

Comment:
If this is only relevant to H7 MCUs, could we guard it in `#if defined(STM32H7)` ?

Reply:
done.